src/overlays/actors/ovl_En_In/z_en_in.c

```diff
@@ -597,7 +597,7 @@ void func_80A79FB0(EnIn* this, GlobalContext* globalCtx) {
 void func_80A7A304(EnIn* this, GlobalContext* globalCtx) {
     if (this->skelAnime.animation == &D_06015814 || this->skelAnime.animation == &D_0601646C) {
         if (this->skelAnime.curFrame == 8.0f) {
-            func_800F41E0(&this->actor.projectedPos, NA_SE_VO_IN_LASH_0, 2);
+            Audio_PlaySoundRandom(&this->actor.projectedPos, NA_SE_VO_IN_LASH_0, 2);
```

Comment:
```suggestion
            Audio_PlaySoundRandom(&this->actor.projectedPos, NA_SE_VO_IN_LASH_0,
                                  NA_SE_VO_IN_LASH_1 - NA_SE_VO_IN_LASH_0 + 1);
```
to help documenting that the two sfx ids should be in a row?